src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -478,7 +478,7 @@ bool AreLayersSupported(InferenceEngine::CNNNetwork& network, std::string& errMe
                                                    }
                                                } else if (info.isConcat()) {
                                                    if (userWarning && !ValidateConcatAxis(layer, errMessage)) {
-                                                       std::cout << errMessage;
+                                                       ov::intel_gna::log::error() << errMessage;
```

Comment:
if this is a warning and does not present place where execution of task is stopped shouldn't be warning here?